config/environments/test.rb

```diff
@@ -16,7 +16,16 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  # config.eager_load = false
+
+  # This configuration change is needed to permit dMMT and MMT to talk to each
```

Comment:
I would maybe add a 'NOTE:' or 'TODO:' with these comments as an additional reminder to come back to it later.

Reply:
done